src/patterns/find-a-support-centre/examples/results/index.njk

```diff
@@ -0,0 +1,130 @@
+---
+layout: none
+---
+{% extends "styles/page-template/_template.njk" %}
+
+{% from "components/collapsible/_macro.njk" import onsCollapsible %}
+
+{% set pageConfig = {
+    "title": "Census 2021",
+    "theme": "census",
+    "absoluteUrl": pageInfo.url,
+    "headMeta": {
+        "description": "Census 2021 is a UK wide population survey."
+    },
+    "header": {
+        "logoHref": "#",
+        "titleLogo": "census-logo-en",
+        "titleLogoHref": "#",
+        "titleLogoAlt": "Census 2021"
+    },
+    "language": {
+        "allLanguagesUrl": "#",
+        "languages": [
+            {
+                "url": "#",
+                "ISOCode": "en",
+                "text": "English",
+                "buttonAriaLabel": "Language selector. Current language: English",
+                "chooseLanguage": "Choose language",
+                "allLanguages": "All languages",
+                "current": true
+            },
+            {
+                "url": "#",
+                "ISOCode": "cy",
+                "text": "Cymraeg",
+                "buttonAriaLabel": "Dewisydd iaith. Iaith gyfredol: Cymraeg",
+                "chooseLanguage": "Dewiswch iaith",
+                "allLanguages": "Pob iaith",
+                "current": false
+            }
+        ]
+    }
+} %}
+
+{% block main %}
+
+    <h1 class="u-mt-l u-fs-xxl">Support centres near <em class="highlight">S1 1XN</em></h1>
+
+    <!-- Result 1 -->
+    <div class="u-pb-m u-pt-m">
+
+        <h2 class="u-fs-l u-mb-xs">Sheffield Central Library</h2>
+        <span class="u-mb-s tag">0.7 miles away</span>
+
+        <p>First Floor, Central Library <br> Surrey Street <br> Sheffield <br> S1 1XZ</p>
+
+        <p>Find us on <a class="icon--external-link" href="https://goo.gl/maps/X48tjZkXA4RaN3LL7" target="_blank" rel="noopener nofollow">Google Maps</a> <br> Telephone: 01142 734 712</p>
+
+        {% set centreDetails %}                          
```

Comment:
Is there any purpose to the centreDetails block? As it's not a template that gets extended I don't think this is doing anything.

Reply:
This has been modified.